src/main/java/org/apache/commons/lang3/StringEscapeUtils.java

```diff
@@ -372,49 +333,6 @@ public int translate(final CharSequence input, final int index, final Writer out
             new NumericEntityUnescaper()
         );
 
-    /**
-     * Translator object for unescaping escaped Comma Separated Value entries.
-     *
-     * While {@link #unescapeCsv(String)} is the expected method of use, this
-     * object allows the CSV unescaping functionality to be used
-     * as the foundation for a custom translator.
-     *
-     * @since 3.0
-     */
-    public static final CharSequenceTranslator UNESCAPE_CSV = new CsvUnescaper();
-
-    static class CsvUnescaper extends CharSequenceTranslator {
```

Comment:
This is a big -1. You cannot break binary compatibility. Run 'mvn' and watch for build failures.